### PR TITLE
Add comment and capitalize message

### PR DESCRIPTION
**Pull request by alamiiin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,2 @@
-console.log('Hello world')
\ No newline at end of file
+// This is my first code to dominate the world
```
Comment (alamiiin) [inline index.js]: Please change "my" to "our"

**Review by alamiiin — COMMENTED** [review]
Looks good so far, but please make the changes I've hinted.

File: index.js (hunk 1 above)
Comment (kpregmi1) [reply index.js]: > Please change "my" to "our"

Okay

**Review by zahidhasandipu — APPROVED** [review]
Approved!

**Review by kpregmi1 — APPROVED** [review]

**Review by Pandey2051 — APPROVED** [review]

**Pull request merged**